Readme.md

```diff
@@ -7,7 +7,7 @@
 </p>
 
 # Fixkture
-Fixkture is an Intellij Plugin to generate test fixtures that supports both Java and Kotlin. It helps generating complex objects from a target class, which may be long and exhausting to do manually, especially for big objects. It does so by providing arbitrary parameters in the object's constructor.
+Fixkture is an Intellij Plugin to generate test fixtures that support both Java and Kotlin. It helps generate complex objects from a target class, which may be long and exhausting to do manually, especially for big objects. It does so by providing arbitrary parameters in the object's constructor.
```

Comment:
It's actually "supports" for the first one. 

```suggestion
Fixkture is an Intellij Plugin to generate test fixtures that supports both Java and Kotlin. It helps generate complex objects from a target class, which may be long and exhausting to do manually, especially for big objects. It does so by providing arbitrary parameters in the object's constructor.
```

Reply:
Indeed the verb `to support` does apply to the noun `plugin` and not to the said `fixtures` in this context. Changes committed. 👍